docs/source/community/contribution_guide.rst

```diff
@@ -287,7 +294,7 @@ Frequently asked questions
    tasks or pull requests with your environment details is helpful and
    appreciated.
 -  **CI tests failed, what does it mean?** Maybe you need to merge with
-   master or rebase with latest changes. Pushing your changes should
+   master or rebase with the latest. Pushing your changes should
```

Comment:
I think a better phrasing would be: "Maybe your PR is based off a broken master, you can try to rebase your change on top of latest master. You can also see the current status of master's CI at https://hud.pytorch.org/"

Reply:
Yup - love it. @svekars can you make the change?

Reply:
Yup, done.